pkg/sql/sem/builtins/generator_builtins.go

```diff
@@ -67,6 +68,12 @@ var Generators = map[string][]tree.Builtin{
 			makeSeriesGenerator,
 			"Produces a virtual table containing the integer values from `start` to `end`, inclusive, by increment of `step`.",
 		),
+		makeGeneratorBuiltin(
+			tree.ArgTypes{{"start", types.Timestamp}, {"end", types.Timestamp}, {"step", types.Interval}},
+			seriesTSValueGeneratorType,
+			makeTSSeriesGenerator,
+			"Produces a virtual table containing the timestamp values from `start` to `end`, inclusive, by increment of `step`.",
+		),
```

Comment:
Can you add a second overload for TimestampTZ, TimestampTZ, Interval -> TimestampTZ?

Reply:
ok

Reply:
It seems tz not implement for yacc.
in `sql.y`

```
| a_expr AT TIME ZONE a_expr %prec AT { return unimplemented(sqllex, "at tz") }
```